src/main/java/com/jmb/springfactory/model/dto/ProductionScheduleDto.java

```diff
@@ -0,0 +1,20 @@
+package com.jmb.springfactory.model.dto;
+
+import java.util.Date;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class ProductionScheduleDto extends BaseDto {
+
+  private static final long serialVersionUID = -8477427436054169331L;
+
+  private String name;
+  private Date estimatedStartDate;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "estimatedStartDate" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)